tests/test_SklearnPipelineWithinPipeline.py

```diff
@@ -62,7 +63,8 @@ def test_pipeline_pca_pipeline_none_lin(self):
                                      [('input', FloatTensorType(data.shape))])
         self.assertTrue(model_onnx is not None)
         dump_data_and_model(data, model, model_onnx,
-                            basename="SklearnPipelinePcaPipelineMinMaxLogReg")
+                            basename="SklearnPipelinePcaPipelineMinMaxLogReg",
+                            allow_failure="StrictVersion(onnxruntime.__version__) <= StrictVersion('0.2.1')")
```

Comment:
instead of add this conditional check everywhere, should we group these tests into sub-folder? it looks the most sklearn converter has to be opset 9 now.

Reply:
Let's add a new issue then?